src/policy.csv

```diff
@@ -0,0 +1,2 @@
+p, u0851e3b-156b-4b7f-85e9-48f0953a6cc8, session, CreateMeetingSession
```

Comment:
can we remove this ? If not, can we keep it as something generic,like CreateUser

Reply:
This file is used to write policy for the current request. So, it gets re-written every time a request is made.

So, just emptying it for now.